V/very_nauty/build_tarballs.jl

```diff
@@ -0,0 +1,49 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "very_nauty"
+version = v"1.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/JuliaGraphs/very_nauty.git", "d6d28d3ad92f86953e7523749e1468ac4a936603")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/very_nauty/
+mkdir -p ${libdir}
+cc -shared -fPIC -Wall -O3 -o "${libdir}/libvn_graph.${dlext}" vn_graph.c
+cp vn_graph.h ${includedir}
```

Comment:
You're creating a file named `${includedir}` which is most likely not what you meant
```suggestion
install -Dvm 644 vn_graph.h -t ${includedir}
```